allowance_unece/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Decodio Applications (<http://decod.io>)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Base UBL allowance",
+    'version': "12.0.1.0.0",
+    "author": "Decodio Applications, Odoo Community Association (OCA)",
+    "development_status": "RC",
+    "summary": "Discount codes for UBL",
```

Comment:
Is this correct?

Reply:
as for discount/allowance codes .. yes.. 
defined by unece, comp'lete codes list here:
http://www.unece.org/trade/untdid/d97b/uncl/uncl5189.htm

will make other requested modifications and push soon